client/packages/requisitions/src/ResponseRequisition/DetailView/ReponseStats/RequestStoreStats.tsx

```diff
@@ -0,0 +1,235 @@
+  label: string;
+  colour: string;
+}) => {
+  const formatNumber = useFormatNumber();
+  if (value === 0) return null;
+
+  const flexBasis = Math.min(Math.round((100 * value) / total), 100);
+
+  return (
+    <>
+      <Tooltip title={`${label}: ${formatNumber.round(value)}`} placement="top">
+        <Box flexBasis={`${flexBasis}%`} flexGrow={1}>
+          <Box sx={{ backgroundColor: colour, height: '20px' }} />
+          <Box
+            style={{
+              textAlign: 'end',
+              paddingRight: 10,
+              paddingTop: 10,
+            }}
+          >
+            {flexBasis > MIN_FLEX_BASIS_TO_SHOW_LABEL ? (
+              <Typography
+                fontSize={12}
+                style={{ textOverflow: 'ellipsis', height: 20 }}
+              >
+                {label}
+              </Typography>
+            ) : null}
+            {flexBasis > MIN_FLEX_BASIS_TO_SHOW_VALUE ? (
+              <Typography fontSize={12}>{formatNumber.round(value)}</Typography>
+            ) : null}
+          </Box>
+        </Box>
+      </Tooltip>
+      <Divider />
+    </>
+  );
+};
+
+const MonthlyConsumption = ({
+  month,
+  flexBasis,
+  averageMonthlyConsumption,
+  showText,
+}: {
+  month: number;
+  flexBasis: string;
+  averageMonthlyConsumption: number;
+  showText: boolean;
+}) => {
+  const t = useTranslation('common');
+  const formatNumber = useFormatNumber();
+  const text = ` (${month} ${t('label.months', {
+    count: month,
+  })})`;
+  const label = `${formatNumber.round(averageMonthlyConsumption * month)}${
+    showText ? text : ''
+  }`;
+
+  return <MonthlyBar flexBasis={flexBasis} label={label} />;
+};
+
+const MonthlyBar = ({
+  label,
+  left,
+  flexBasis,
+}: {
+  label: string;
+  left?: boolean;
+  flexBasis?: string;
+}) => {
+  const directionStyle = left
+    ? { borderLeftWidth: 1, paddingLeft: 8 }
+    : { paddingLeft: 3, paddingRight: 8, borderRightWidth: 1 };
+  return (
+    <Box
+      sx={{
+        borderWidth: 0,
+        borderBottomWidth: 1,
+        borderColor: 'gray.dark',
+        borderStyle: 'solid',
+        height: '20px',
+        overflow: 'hidden',
+      }}
+      style={{ ...directionStyle, textAlign: left ? undefined : 'right' }}
+      flexBasis={flexBasis}
+    >
+      <Tooltip title={label} placement="top">
+        <Typography
+          variant="body1"
+          fontSize={12}
+          style={{ color: 'gray.dark' }}
+        >
+          {label}
+        </Typography>
+      </Tooltip>
+    </Box>
+  );
+};
+
+const CalculationError = ({
+  isAmcZero,
+  isSohAndQtyZero,
+}: {
+  isAmcZero?: boolean;
+  isSohAndQtyZero?: boolean;
+}) => {
+  const t = useTranslation('replenishment');
+  const detail = isAmcZero
+    ? `: ${t('error.amc-is-zero')}`
+    : isSohAndQtyZero
+    ? `: ${t('error.soh-and-suggested-quantity-are-zero')}`
+    : '';
+  const message = `${t('error.unable-to-calculate')}${detail}`;
+
+  return (
+    <Box display="flex" padding={1} gap={1}>
```

Comment:
```suggestion
    <Box display="flex" padding={2} gap={1}>
```

The error is jammed up against the edge.. a little more padding would be good

<img width="643" alt="Screenshot 2022-11-30 at 9 25 48 AM" src="https://user-images.githubusercontent.com/9192912/204641543-b8abe014-d20d-476c-b4c8-b0f8a01d15ba.png">